src/Components/MailtoDropdown.tsx

```diff
@@ -130,7 +133,7 @@ export const MailtoDropdown = (props: MailtoDropdownProps) => {
     const handlers =
```

Comment:
Maybe it is smart to memoize these.

Reply:
I mean, we can… But I really struggle to decide when that's a good idea, especially since you barely used `useCallback()` in your PR at all, even for stuff where I definitely would have. :D 

In this case, computing `handlers` requires ~~ten~~five simple equality checks on strings. I wouldn't be surprised if the memoization was more complicated than that.